packages/suite/src/components/suite/AppNavigation/index.tsx

```diff
@@ -71,7 +82,7 @@ const AppNavigation = ({ items }: Props) => {
                     <StyledNavLink
                         key={route}
                         active={active}
-                        onClick={() => goto(route, undefined, true)}
+                        onClick={() => goto(route, params, true)}
```

Comment:
just use selectedAccount.params here (it will be either object or undefined) and it will not break anything

this way all above checks could be removed

Reply:
It is. If you don't check the router goto function will add one new hash per click. Like this. 

![image](https://user-images.githubusercontent.com/3112191/94567175-76092880-026b-11eb-9aa5-6ec87eea9fcf.png)

Reply:
it's because of third param `true` (preserveParams in goto)

so it should be like:
```
const { params } = selectedAccount;
params ? goto(route, params) : goto(route, undefined, true)
```